<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Scaletech-GS" failures="1" tests="3" name="Think.ihk_test.dashboard" time="24.606" errors="0" timestamp="2024-01-29T18:58:30 IST" skipped="1">
  <testcase classname="Think.ihk_test.dashboard" name="motifsImageVisibleFinal" time="14.701">
    <skipped/>
  </testcase> <!-- motifsImageVisibleFinal -->
  <system-out/>
  <testcase classname="Think.ihk_test.dashboard" name="motifsNameVisiblefinal" time="2.950"/>
  <system-out/>
  <testcase classname="Think.ihk_test.dashboard" name="motifsImageVisibleFinal" time="6.955">
    <failure message="Lists differ at element [0]: https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/stage-check-111_preview.jpg != https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/motif-preview-placeholder.jpg expected [https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/stage-check-111_preview.jpg] but found [https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/motif-preview-placeholder.jpg]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Lists differ at element [0]: https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/stage-check-111_preview.jpg != https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/motif-preview-placeholder.jpg expected [https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/stage-check-111_preview.jpg] but found [https://s3.eu-central-1.amazonaws.com/ads.ihk-stage/assets/motifs/motif-preview-placeholder.jpg]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:1103)
at org.testng.Assert.assertEquals(Assert.java:1065)
at Think.ihk_test.dashboard.motifsImageVisibleFinal(dashboard.java:117)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- motifsImageVisibleFinal -->
  <system-out/>
</testsuite> <!-- Think.ihk_test.dashboard -->
